src/analysis/fourier/add_circle.lean

```diff
@@ -326,4 +348,60 @@ begin
   ring,
 end
 
+/-- The Fourier coefficients of a continuous function `ℝ / ℤ • T → ℂ`. -/
+def continuous_map.fourier_coeff (f : C(add_circle T, ℂ)) (n : ℤ) : ℂ :=
```

Comment:
can you mention both this definition and the main theorem below (has_sum_fourier_series_of_summable) in the file-level docstring?